### ISSUE TITLE
Large JSON files

### ISSUE THREAD

**Issue by mbenezra** [VisualDataWeb/WebVOWL#156]
Is there a limit to the size of the JSON file that can be uploaded to WebVOWL? I tried uploading a file of 160MB, is that too large?

**Comment by christopheralcock** [comment]
is this a 160MB ontology?

**Comment by mbenezra** [comment]
Yes. I used OWL2VOWL to produce the JSON file from an OWL file produced by https://github.com/IHTSDO/snomed-owl-toolkit from the SNOMED CT RF2 file format.

**Comment by mbenezra** [comment]
SNOEMD CT has approximately 350,000 terms.

**Comment by vitalis-wiens** [comment]
Yes, there is a limit to the file size but only in the public version of WebVOWL (owl2vowl) 

Since you already have the JSON file, you can use a local instance of WebVOWL that does not have this file size restriction (http://downloads.visualdataweb.de/webvowl_1.1.4.zip)

However, due to the size and number of elements you might have some performance problems.

Kind Regards, 
Vitalis

**Comment by madansr7** [comment]
I have a 135kb file, that has the following stats. The file loads but the UI is extremely slow to process at Scale level "0" when data type and object properties are both shown. Is there a workaround to speed things up?

Classes: 320
Object prop.: 420
Datatype prop.: 2257
Individuals: 0
Nodes: 2577
Edges: 2793

**Comment by madansr7** [comment]
any thoughts or updates?
